build.html

```diff
@@ -1,61 +1,33 @@
 <script type="text/javascript">
-  var DynamicListFuncs = {};
-  var dynamicLists = {};
+  var DynamicListFuncs = DynamicListFuncs || {};
+  var dynamicLists = dynamicLists || {};
 </script>
 {{#if layout}}
   {{#if advancedSettings.cssEnabled}}
   <style>
   {{{advancedSettings.cssCode}}}
   </style>
   {{/if}}
-  <div data-dynamic-lists-id="{{ id }}" data-dynamic-lists-layout="{{ layout }}" data-dynamic-lists-uuid="{{uuid}}"></div>
+  <div data-dynamic-lists-1-2-0-id="{{ id }}" data-dynamic-lists-layout="{{ layout }}" data-dynamic-lists-uuid="{{uuid}}"></div>
   {{#if advancedSettings.jsEnabled}}
   <script type="text/javascript">
     Fliplet().then(function () {
       var id = {{ id }};
       var data = Fliplet.Widget.getData(id);
+
       try {
         {{{ advancedSettings.jsCode }}}
       } catch (e) {
         console.log('Error with custom JS code', e);
       }
+      var DynamicList = Fliplet.Registry.get('dynamic-list:1.2.0:{{ layout }}:{{ id }}');
```

Comment:
I would recommend using the widget `uuid` instead of `id`, as the published app doesn't have the same id